src/codegen/replacements.ts

```diff
@@ -120,7 +120,7 @@ for (const name in enums) {
   if (typeof value === null) throw new Error("Invalid enum object " + name + " defined in " + import.meta.file);
   const keys = Array.isArray(value) ? value : Object.keys(value).filter(k => !k.match(/^[0-9]+$/));
   define[`$${name}IdToLabel`] = "[" + keys.map(k => `"${k}"`).join(", ") + "]";
-  define[`$${name}LabelToId`] = "{" + keys.map(k => `"${k}": ${keys.indexOf(k)}`).join(", ") + "}";
+  define[`$${name}LabelToId`] = "{" + keys.map(k => `"${k}": ${keys.indexOf(k) + 1}`).join(", ") + "}";
```

Comment:
what's this change?

Reply:
This was a bug me and @paperdave found related to associating loader string value to number value

Reply:
Can you add a comment with example of what the output should be just so it's easier for future readers to understand what it's doing?